isstools/xview.py

```diff
@@ -239,9 +239,9 @@ def selectBinnedDataFilesToPlot(self):
     def plotBinnedData(self):
         selected_items = (self.listFiles_bin.selectedItems())
         self.figureBinned.ax.clear()
-        self.toolbar._views.clear()
-        self.toolbar._positions.clear()
-        self.toolbar._update_view()
+        #self.toolbar._views.clear()

```

Comment:
@elistavitski how about replacing these three lines with:
`self.toolbar.clear()`

does that resolve it? I'm not familiar with this, we may all have to read documentation and debug.

Reply:
Nope that removes the toolbar :)

Reply:
hmm, i think the best option is to call:
`xview.toolbar.`
and hit TAB and see what options you have
worst case scenario:
`xview.toolbar._` 
and then hit TAB.

The issue here is that `toolbar._views` has changed in a more recent version of matplotlib making the usage incompatible. If we can't solve this, we can try downgrading matplotlib.

Would you mind if i installed a VNC server on one of your workstations and tested it out quickly?
The VNC server would be secure (the machine won't accept requests outside)
I can also optionally allow others to view what im doing (but they can't interact with the window)

does this sound like a good way to proceed?

Reply:
I also ping @tacaswell here as he might per chance know what changes have been made with respect to `._views`